src/Chapter1.hs

```diff
@@ -620,8 +636,13 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Integral a => a -> a
+sumLast2 n = tens n + ones n
+    where
+      tens :: Integral a => a -> a
+      tens t = div (mod (abs t) 100) 10
+      ones :: Integral a => a -> a
+      ones o = mod (abs o) 10
```

Comment:
💯 perfect usage of standard `div` and `mod`s!

You can see now that `ones` is actually the `lastDigit` function from the previous solution 🙂

Reply:
Yeah, I was really solving each of these in isolation. It didn't really occur to me that I could reuse lastDigit here, but that of course makes sense that I would be able to use it (I was using it in the REPL obviously!).